Removing reader/writer customization

ewmassey opened the pull request:
Removing reader and writer customizing framework in favor of monkey patching for now.

rgbkrk commented on papermill/execute.py:
```diff
@@ -44,8 +44,8 @@ def _parameterize_notebook(notebook_path, output_path, parameters):
 
-    # Remove the first cell of the block and replace it with the param content.
-    nb = read_notebook(notebook_path)
+    # Find `parameters` cell and replaced with our parameters.
```
...and replace with...

ewmassey replied:
I had two functions called read_notebook. One read a ipynb file and
returned a NotebookNode. The other is the API function that returns a
Papermill object. I removed the former to avoid confusion.

rgbkrk replied:
My comment here was to fix the grammar. :)

ewmassey commented:
I had two functions called read_notebook. One read a ipynb file and
returned a NotebookNode. The other is the API function that returns a
Papermill object. I removed the former to avoid confusion.

On Wed, Aug 2, 2017 at 7:19 PM Kyle Kelley <notifications@github.com> wrote:

> *@rgbkrk* requested changes on this pull request.
>
> What's the motivation for removing the read_notebook function name?
> ------------------------------
>
> In papermill/execute.py
> <https://github.com/nteract/papermill/pull/16#discussion_r131040585>:
>
> > @@ -44,8 +44,8 @@ def _parameterize_notebook(notebook_path, output_path, parameters):
>              val = '"%s"' % val  # TODO: Handle correctly escaping input strings.
>          param_content += '%s = %s\n' % (var, val)
>
> -    # Remove the first cell of the block and replace it with the param content.
> -    nb = read_notebook(notebook_path)
> +    # Find `parameters` cell and replaced with our parameters.
>
> ...and replace with...
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/nteract/papermill/pull/16#pullrequestreview-53980071>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ABRh2_vPatW-wJ_GngbtTGeULM9heheDks5sUS4ugaJpZM4Orsf_>
> .
>
